core/players-handbook-2024/classes/class-cleric.xml

```diff
@@ -0,0 +1,447 @@
+			<stat name="cleric:spellcasting:slots:2" value="1" level="4" />
+			<stat name="cleric:spellcasting:slots:3" value="2" level="5" />
+			<stat name="cleric:spellcasting:slots:3" value="1" level="6" />
+			<stat name="cleric:spellcasting:slots:4" value="1" level="7" />
+			<stat name="cleric:spellcasting:slots:4" value="1" level="8" />
+			<stat name="cleric:spellcasting:slots:4" value="1" level="9" />
+			<stat name="cleric:spellcasting:slots:5" value="1" level="9" />
+			<stat name="cleric:spellcasting:slots:5" value="1" level="10" />
+			<stat name="cleric:spellcasting:slots:5" value="1" level="18" />
+			<stat name="cleric:spellcasting:slots:6" value="1" level="11" />
+			<stat name="cleric:spellcasting:slots:6" value="1" level="19" />
+			<stat name="cleric:spellcasting:slots:7" value="1" level="13" />
+			<stat name="cleric:spellcasting:slots:7" value="1" level="20" />
+			<stat name="cleric:spellcasting:slots:8" value="1" level="15" />
+			<stat name="cleric:spellcasting:slots:9" value="1" level="17" />
+
+			<stat name="cleric:spellcasting:prepare" value="4" level="1" />
+			<stat name="cleric:spellcasting:prepare" value="1" level="2" />
+			<stat name="cleric:spellcasting:prepare" value="1" level="3" />
+			<stat name="cleric:spellcasting:prepare" value="1" level="4" />
+			<stat name="cleric:spellcasting:prepare" value="2" level="5" />
+			<stat name="cleric:spellcasting:prepare" value="1" level="6" />
+			<stat name="cleric:spellcasting:prepare" value="1" level="7" />
+			<stat name="cleric:spellcasting:prepare" value="1" level="8" />
+			<stat name="cleric:spellcasting:prepare" value="2" level="9" />
+			<stat name="cleric:spellcasting:prepare" value="1" level="10" />
+			<stat name="cleric:spellcasting:prepare" value="1" level="11" />
+			<stat name="cleric:spellcasting:prepare" value="1" level="13" />
+			<stat name="cleric:spellcasting:prepare" value="1" level="15" />
+			<stat name="cleric:spellcasting:prepare" value="1" level="17" />
+			<stat name="cleric:spellcasting:prepare" value="1" level="18" />
+			<stat name="cleric:spellcasting:prepare" value="1" level="19" />
+			<stat name="cleric:spellcasting:prepare" value="1" level="20" />
+
+
+			<select type="Spell" name="Cantrip (Cleric)" supports="$(spellcasting:list), 0" level="1" number="3" spellcasting="Cleric" />
+			<select type="Spell" name="Cantrip (Cleric)" supports="$(spellcasting:list), 0" level="4" number="1" spellcasting="Cleric" />
+			<select type="Spell" name="Cantrip (Cleric)" supports="$(spellcasting:list), 0" level="10" number="1" spellcasting="Cleric" />
+		</rules>
+	</element>
+
+	<element name="Level 1: Divine Order" type="Class Feature" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_CLASS_FEATURE_CLERIC_DIVINEORDER">
+		<description>
+			<p>You have dedicated yourself to one of the following sacred roles of your choice. You may choose one of the following features to gain:</p>
+			<p class="indent"><i>Protector</i> or <i>Thaumaturge</i></p>
```

Comment:
Probably should list what each item does in the class description. Doesn't need to show on the sheet until you select them.
```suggestion
			<p class="indent"><em><strong>Protector.</strong></em> Trained for battle, you gain proficiency with Martial weapons and training with Heavy armor.</p>
			<p class="indent"><em><strong>Thaumaturge.</strong></em> You know one extra cantrip from the Cleric spell list. In addition, your mystical connection to the divine gives you a bonus to your Intelligence (Arcana or Religion) checks. The bonus equals your Wisdom modifier (minimum of +1).</p>
```

Reply:
With how the <div element="..."> section is set up, the options follow the Divine Order entry directly with their own description, and as such the Divine Order needn't expand on the content of the options

Reply:
I wasn't actually seeing them in the description of the app. My system could've just been borked, though.

That being said, I've always had issues with DIV elements actually showing up in the sidebar description for some reason.